tox.ini

```diff
@@ -3,7 +3,9 @@ envlist=py{36,37,38,39,310}-test,flake8,check_readme,check-dist,py{37}-selfcheck
 
 [pytest]
 testpaths=testing
-filterwarnings=error
+filterwarnings=
+    error
+    ignore:.*tool\.setuptools_scm.*
```

Comment:
Wouldn't it be better to use `with pytest.warns`? Don't you want to ensure the warning are showing up, and be alerted if they are not?

Reply:
only in the tests for them